nrich-notification/README.md

```diff
@@ -33,33 +46,76 @@ To be able to use this module following configuration is required:
     public NotificationResponseService notificationResponseService(NotificationResolverService notificationResolverService) {
         return new WebMvcNotificationResponseService(notificationResolverService);
     }
-}
 
+}
 ```
 
-`ConstraintConversionService` is a service responsible for converting `javax.validation.ConstraintViolation` list to Spring `Errors`
-instances making implementation of `NotificationResolverService` less complex since it only needs to work with Spring's `Errors` when processing validation errors.
+### [ConstraintConversionService][constraint-conversion-service-url]
+
+[`ConstraintConversionService`][constraint-conversion-service-url] is a service responsible for converting `javax.validation.ConstraintViolation` list to Spring `Errors` instance.
```

Comment:
Not sure about this `is a service` part. I think we can omit it. I think "`ConstraintConversionService` is responsible..." is clear enough. We have this for other service classes.

Reply:
I changed everywhere where it seem extra, but there is a `is the core service` part of the sentence for `NotificationResolverService` which I would leave to signal its importance.